code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1">
+
+<head>
+  <script src="script-math.js"></script>
+  <script src="script-else.js"></script>
+  <link rel="stylesheet" id="pagestyle" type="text/css" href="theme-default.css">
```

Comment:
id-page-style

Reply:
renamed and unified with the rest of the code id="id-page-style"... will be in next commit